### DISCUSSION TITLE
"This video may be inappropriate for some users"

### DISCUSSION THREAD

**Discussion by ghost** [FreeTubeApp/FreeTube#5940] [Q&A] [answered]
This issue is now arising on some videos (use https://youtu.be/ZM8zdby8sGA as an example). 

But i am forced to view this on YT and with an acct.

**Comment by absidue** [comment, marked as answer]
YouTube fixed the only known age-restriction bypass, so yes now the only way to watch age-restricted videos is by using the official YouTube apps or website while logged into an age-verified YouTube account.
